var/spack/repos/builtin/packages/go-md2man/package.py

```diff
@@ -6,41 +6,14 @@
 from spack.package import *
 
 
-class GoMd2man(Package):
+class GoMd2man(GoPackage):
     """go-md2man converts markdown into roff (man pages)"""
 
     homepage = "https://github.com/cpuguy83/go-md2man"
     url = "https://github.com/cpuguy83/go-md2man/archive/v1.0.10.tar.gz"
 
     license("MIT")
 
+    version("2.0.3", sha256="7ca3a04bb4ab83387538235decc42a535097a05d2fb9f2266d0c47b33119501f")
     version("2.0.2", sha256="2f52e37101ea2734b02f2b54a53c74305b95b3a9a27792fdac962b5354aa3e4a")
     version("1.0.10", sha256="76aa56849123b99b95fcea2b15502fd886dead9a5c35be7f78bdc2bad6be8d99")
-
-    depends_on("go")
-
-    resource(
-        name="blackfriday",
-        url="https://github.com/russross/blackfriday/archive/v1.5.2.tar.gz",
-        sha256="626138a08abb8579474a555e9d45cb5260629a2c07e8834428620a650dc9f195",
-        placement="blackfriday",
-        destination=join_path("src", "github.com", "russross"),
-    )
-
-    def patch(self):
-        mkdirp(join_path(self.stage.source_path, "src", "github.com", "russross"))
-
-        mkdirp(join_path(self.stage.source_path, "src", "github.com", "cpuguy83"))
-
-        ln = which("ln")
-        ln("-s", self.stage.source_path, join_path("src", "github.com", "cpuguy83", "go-md2man"))
-
-    def install(self, spec, prefix):
-        with working_dir("src"):
-            env["GOPATH"] = self.stage.source_path
-            env["GO111MODULE"] = "off"
```

Comment:
This doesn't appear in the base class.  Is there any idea if it's guaranteed to always be off?

Reply:
It really doesn't need to be off. I don't know why it was set that way before. Perhaps a legacy go configuration that no longer makes sense as Go Modules are standard now in all of the versions of Go we have in Spack.